app/scripts/scrollHelp.js

```diff
@@ -117,8 +119,18 @@ ipcRenderer.on('extensionStatesReply', (event, payload) => {
 	}
 
 	if (payload.scrollHelpPosition) {
-		scrollHelpPosX = payload.scrollHelpPosition.left;
-		scrollHelpPosY = payload.scrollHelpPosition.top;
+		const { innerWidth: windowWidth, innerHeight: windowHeight } = window;
+		const left = parseInt(payload.scrollHelpPosition.left.replace('px', ''));
+		const top = parseInt(payload.scrollHelpPosition.top.replace('px', ''));
```

Comment:
Graag bij `parseInt` ook altijd een radix meegeven. Zo is `parseInt('0xF')` niet hetzelfde als `parseInt('0xF', 10)`
Zie: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt

Reply:
Aangezien wij bijna altijd met decimals werken gebruiken we altijd die radix 10.

Reply:
En misschien niet heel die replace in de parseInt steken. Maakt het wat onleesbaar

Reply:
Dus via een radix geef je aan of de uitkomst een decimaal, hexadecimaal, .... moet zijn?

Reply:
Via de radix geef je aan wat je wil bekomen. Dus hoe hij het met omzetten